src/wp-admin/includes/class-wp-debug-data.php

```diff
@@ -839,6 +839,21 @@ public static function debug_data() {
 			);
 		}
 
+		// Server time.
+		$date = new \DateTime( 'now', new \DateTimeZone( 'UTC' ) );
+		$info['wp-server']['fields']['current'] = array(
+			'label' => __( 'Current time' ),
+			'value' => $date->format( \DateTime::ATOM ),
```

Comment:
Let's remove the `\` from `\DateTime` in the PR. We don't use it elsewhere in Core

Reply:
Done. Thanks ;)